src/js/utils.js

```diff
@@ -114,16 +114,16 @@ function explodeSubdomains(fqdn, all) {
  */
 function estimateMaxEntropy(str) {
   /*
-   * Don't process item + key's longer than LOCALSTORAGE_MAX_LEN_FOR_ENTROPY_EST.
-   * Note that default quota for local storage is 5MB and
-   * storing fonts, scripts or images in for local storage for
-   * performance is not uncommon. We wouldn't want to estimate entropy
-   * for 5M chars.
-   */
+  * Don't process item + key's longer than LOCALSTORAGE_MAX_LEN_FOR_ENTROPY_EST.
+  * Note that default quota for local storage is 5MB and
+  * storing fonts, scripts or images in for local storage for
+  * performance is not uncommon. We wouldn't want to estimate entropy
+  * for 5M chars.
+  */
   let MAX_LS_LEN_FOR_ENTROPY_EST = 256;
 
   // common classes of characters that a string might belong to
-  let SEPS = "._-";
+  let SEPS = "._-x";
```

Comment:
Could we add a test for the "x" separator? One of those screen resolution strings from the badger data notebook.

Reply:
Good call -- just added one in